cmd/statusd/whispercfg.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/status-im/status-go/geth/params"
+)
+
+// makeNodeConfig creates node configuration object from flags
+func whisperConfig(nodeConfig *params.NodeConfig) (*params.NodeConfig, error) {
+
+	nodeConfig.ListenAddr = *listenAddr
+
+	nodeConfig.LogLevel = *logLevel
+
+	// TODO this was in wnode-status ... can it be completely removed ?
```

Comment:
If logLevel and logFile are handled in other code already, than it can be safely removed from here.

Reply:
since logLevel and logFile already existed in statusd/main.go ...
I will remove corresponding code that I had a 'TODO' and commented out.